File: Medium/JavaScript/2433. Find The Original Array of Prefix Xor.js
var findArray = function(pref) {
    let arr = Array(pref.length);
    arr[0] = pref[0];

    for(let i = pref.length - 1; 0 < i; i--){
        arr[i] = pref[i] ^ pref[i - 1];
    }

    return arr;
};

File: Medium/Python/2433. Find The Original Array of Prefix Xor.py
class Solution(object):
    def findArray(self, pref):
        lis = [0] * len(pref)
        lis[0] = pref[0]

        for i in range(len(pref) - 1, 0, -1):
            lis[i] = pref[i] ^ pref[i - 1]

        return lis
